rasa/shared/nlu/training_data/lookup_tables_parser.py

```diff
@@ -17,4 +19,6 @@ def add_item_to_lookup_tables(
         existing_lookup_tables.append({"name": title, "elements": [item]})
     else:
         elements = matches[0]["elements"]
+        if not isinstance(elements, list):
```

Comment:
🤔 it sounds like a wrong type annotation then. Will see...

Reply:
did you already revisit this one?

Reply:
no I haven't implemented it yet, wanted to wait for an answer on other points first 👍🏻 will do

Reply:
ok I think I finally get it. The typing is correct because the Markdown reader adds the lookup table file [here](https://github.com/RasaHQ/rasa/blob/8692ea37d03fa4c4244919e0ab310ed1691990a6/rasa/shared/nlu/training_data/formats/markdown.py#L106) (which is maybe questionable, but out of the scope here IMO). But if during parsing we're trying to add an item to this unloaded lookup table file, then it means that the parsing is broken. Implemented this behaviour here: 3db8c33d327

Reply:
Why can this be a `Union[Text, List]` in the first place? What was wrong with the old type annotation?

Reply:
I would answer this question the same way as in my previous comment, that's the way the code works now, the type annotation didn't reflect how the code works now 🤷🏻 if this answer is not good enough, can you detail a bit more what you're looking for? 😬

Reply:
ah, I see,the markdown reader is doing that so the old type annotation was wrong 👍🏻